src/front-end/vue/components/next-pieces.ts

```diff
@@ -12,14 +11,15 @@ export const NextPieces = () => ({
     },
   },
   template: `
-  <div class="${flexGrowShrink} ${flexCol}">
-    <h3>Next:</h3>
-    <bd-block 
-      v-for="(block, i) in preview" 
-      v-bind:block="block"
-      v-bind:key="i"
-    >
-    </bd-block>
+  <div class="ba bw2 b--vue-green mb4 shadow-vue-green">
+    <h4 class="black bg-vue-green f3 mb0 mt0 tc">NEXT</h4>
```

Comment:
I just noticed this was previously a `<h3>`. I didn't change it to h4 for any particular reason, I can change it back